README.md

```diff
@@ -426,6 +411,14 @@ scrollable will default to using the window for the scroll binding.
 
 You can optionally pass an offset value.   This value will be used when calculating if the bottom of the scrollable has been reached.  
 
+* **eventDebounce**
+
+```hbs
+{{infinity-loader eventDebounce=debounceTimer}}
+```
+
+Default is 10ms.  You can optionally pass a debounce time to delay loading the list when reach bottom of list
```

Comment:
10ms is super fast for a debounce, specially for a lib that is just "detecting being in viewport" (ie - not an interpolated animation or something that needs rapid event streams).  Should we make it like 50 - 100?

Reply:
You can try it out on 4200 @ 50 ms.